stash_engine/app/controllers/stash_engine/resources_controller.rb

```diff
@@ -54,10 +58,16 @@ def create
       resource.save
       resource.fill_blank_author!
       redirect_to metadata_entry_pages_find_or_create_path(resource_id: resource.id)
-    rescue StandardError
+    rescue StandardError => error
+      log.error("Unable to create new resource: #{to_backtrace(error)}")
```

Comment:
I think you can just change this to `logger` instead of `log` and it should already exist because Rails mostly just leaves it sitting around for you to use in MVC classes without any special extra work.